BUILD.bazel

```diff
@@ -295,6 +295,7 @@ cc_library(
         ],
     }),
     hdrs = [
+        "src/ray/object_manager/plasma/allocator.h",
```

Comment:
Do we need this for the plasma client? (I cannot find any change from client.cc )